notebooks/matrix_tube_pipeline_seqcount_norm.ipynb

```diff
@@ -1186,89 +1537,124 @@
     "names = make_2D_array(plate_df, data_col=PM_SAMPLE_KEY, well_col=well_col)\n",
     "i5 = make_2D_array(plate_df, data_col='i5 name', well_col=well_col)\n",
     "i7 = make_2D_array(plate_df, data_col='i7 name', well_col=well_col)"
-   ],
-   "outputs": [],
-   "execution_count": null
+   ]
   },
   {
    "cell_type": "code",
-   "metadata": {},
-   "source": "plot_plate_vals(concs, color_map='viridis')",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-03-04T00:19:33.764655Z",
+     "start_time": "2025-03-04T00:19:31.458Z"
+    }
+   },
    "outputs": [],
-   "execution_count": null
+   "source": [
+    "plot_plate_vals(concs, color_map='viridis')"
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
-   "source": "#### Plate maps for individual constituent plates"
+   "source": [
+    "#### Plate maps for individual constituent plates"
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
-   "source": "##### Library concentration heatmap, Plate 1"
+   "source": [
+    "##### Library concentration heatmap, Plate 1"
+   ]
   },
   {
    "cell_type": "code",
-   "metadata": {},
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-03-04T00:19:35.903480Z",
+     "start_time": "2025-03-04T00:19:34.414882Z"
+    }
+   },
+   "outputs": [],
    "source": [
     "plot_plate_vals(concs[np.ix_(EVEN_ROWS,EVEN_COLS)],\n",
```

Comment:
the `np.ix_` use is here as well

Reply:
...and below, I'll stop commenting on it

Reply:
:D